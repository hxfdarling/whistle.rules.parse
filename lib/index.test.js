/* eslint-disable arrow-parens */
const parse = require('./index');

describe('test rules', () => {
  it('parse', async () => {
    const result = parse(`
//baidu.com 127.0.0.1:8080
/web/token https://qq.com/web/token
qq.com 127.0.0.1:8080/web
//nohost.bytedance.net/web 127.0.0.1:8080/web
fudao.qq.com file:///User/zman/qq
`);
    Object.keys(result).forEach((key) => {
      const t = result[key];
      Object.keys(t).forEach((key2) => {
        console.log(t[key2].rawPattern);
      });
    });
    expect('module').toMatchSnapshot('module');
  });
});
