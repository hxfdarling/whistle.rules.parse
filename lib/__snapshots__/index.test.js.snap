// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test rules parse: module 1`] = `"module"`;
